scripts/kconfig/kconfigfunctions.py

```diff
@@ -335,11 +335,47 @@ def dt_compat_enabled(kconf, _, compat):
     return "n"
 
 
+def dt_compat_get_string(kconf, _, compat, inst=0, ret_def='', prop='label'):
```

Comment:
Call it `get_str` instead of `get_string`, for consistency with the other functions. I wonder if `dt_str_from_compat` might be nice too. Then there could be a `dt_str_from_path` later on if needed.

A signature like `(kconf, _, prop, compat, inst, default="")` might be better, where you're forced to pass everything but the default value. Makes it a bit more explicit.

Use double quotes instead of single quotes, just for consistency with the rest of the code (I know it's less common in Python).

Could call it `default` instead of `ret_def` maybe.

Reply:
I did all changes except name signature to **dt_str_from_compat**. I preferred keep the actual method name convention on the file since this was recently introduced. However, I don't have any problem to change it.